provider/maas/environ.go

```diff
@@ -286,36 +253,13 @@ func (env *maasEnviron) SetCloudSpec(_ stdcontext.Context, spec environscloudspe
 		return errors.Trace(err)
 	}
 
-	// We need to know the version of the server we're on. We support 1.9
-	// and 2.0. MAAS 1.9 uses the 1.0 api version and 2.0 uses the 2.0 api
-	// version.
 	apiVersion := apiVersion2
```

Comment:
This feels like it can also die, and we just inline it.

Reply:
> Very rough first pass. I wonder if we should drop `2` as the suffix for `maas`. I'm inclined just to see maas in the provider.

Yes; as per the patch description I will follow with this. I wanted to keep the diff relatively cogent, then do a round of refactoring for names and such.